### PR TITLE
Adds Missing "using" statements

### LINKED ISSUE #13701: Missing "using" statements
There are several missing "using" statements (System, System.Reflection, System.IO) throughout the documentation beginning at [API info and description](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-2.2&tabs=visual-studio-mac#api-info-and-description). 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 14fce65f-1337-58cb-2193-8d8e0e62792d
* Version Independent ID: 59222460-a971-e76e-4fe9-23d4d7d768ee
* Content: [Get started with Swashbuckle and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-2.2&tabs=visual-studio-mac#add-and-configure-swagger-middleware)
* Content Source: [aspnetcore/tutorials/getting-started-with-swashbuckle.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/getting-started-with-swashbuckle.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @zuckerthoben
* Microsoft Alias: **scaddie**

### PR DESCRIPTION
**Pull request by mattielee**
Fixes #13701
References same file as the InfoClassNamespace. I PR'd that file as well. /tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@mattielee  get the URL from the details link
image

As you can see, the snippet was not imported. 
image


We typically don't tell folks what includes are needed  except for introduction/onboarding tutorials. I'll let @scottaddie  decide if that's the case here.

**Comment by Rick-Anderson** [comment]
It gets high traffic. 
image

Check the open/closed issues to see if anyone has had problems figuring out the includes.

image

**Comment by Rick-Anderson** [comment]
It's not showing up because you need to do both changes in on PR. You can do that from the browser by selecting the **Files changed** tab and selecting the ellipsis ... 

image

Then navigate to the file to edit

image

File: aspnetcore/tutorials/getting-started-with-swashbuckle.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ If using directories with IIS or a reverse proxy, set the Swagger endpoint to a
 
 Swagger provides options for documenting the object model and customizing the UI to match your theme.
 
+In the Startup class, add the following namespaces:
+[!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs?name=snippet_PreReqNamespaces)]
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle.md]: If we accept this, use absolute path, not relative path. That is
`~/tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.Swashbuckle/Startup2.cs`

Comment (scottaddie) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: A region named `snippet_PreReqNamespaces` will need to be defined in the *Startup2.cs* file. This is why the snippet doesn't display.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: > A region named snippet_PreReqNamespaces will need to be defined in the Startup2.cs file. This is why the snippet doesn't display.

Done in PR #13702

Comment (mattielee) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: I added the region to the _Startup2.cs_ file in this PR.

Comment (mattielee) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: Just now

Comment (Rick-Anderson) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: Did you check the build?

Comment (mattielee) [reply aspnetcore/tutorials/getting-started-with-swashbuckle.md]: Just checked, seems to be showing up fine now.

**Pull request merged**